components/layout/block.rs

```diff
@@ -1095,14 +1096,69 @@ impl BlockFlow {
                                                           self.base.position.size);
     }
 
+    pub fn get_explicit_block_containing_size(&self, layout_context: &LayoutContext) -> Option<Au> {
+        if self.is_root() || self.is_fixed() {
+            let screen_size = LogicalSize::from_physical(self.fragment.style.writing_mode,
+                                                         layout_context.shared.screen_size);
+            Some(screen_size.block)
+        } else if self.base.flags.contains(IS_ABSOLUTELY_POSITIONED) {
+            self.base.absolute_cb.get_explicit_block_containing_size(layout_context)
+        } else {
+            self.base.block_container_explicit_block_size
+        }
+    }
+
+    fn get_explict_block_size(&self, containing_block_size: Option<Au>) -> Option<Au> {
```

Comment:
typo: `explict` -> `explicit`

Also I'd remove the `get` from this method and the one above.